institut-francais-darcheologie-orientale.csl

```diff
@@ -1,337 +1,588 @@
 <?xml version="1.0" encoding="utf-8"?>
-<style xmlns="http://purl.org/net/xbiblio/csl" version="1.0" class="note" demote-non-dropping-particle="sort-only" page-range-format="expanded">
+<style class="note" version="1.0" initialize-with="." page-range-format="expanded" demote-non-dropping-particle="sort-only" default-locale="fr-FR" xmlns="http://purl.org/net/xbiblio/csl">
+  <!-- This style was edited with the Visual CSL Editor (https://editor.citationstyles.org/visualEditor/) -->
   <info>
-    <title>Institut français d'archéologie orientale</title>
-    <title-short>IFAO</title-short>
+    <title>Institut français d'archéologie orientale (Français)</title>
+    <title-short>IFAO (FR)</title-short>
     <id>http://www.zotero.org/styles/institut-francais-darcheologie-orientale</id>
     <link href="http://www.zotero.org/styles/institut-francais-darcheologie-orientale" rel="self"/>
-    <link href="http://www.zotero.org/styles/gallia" rel="template"/>
-    <link href="https://www.ifao.egnet.net/publications/publier/normes-ed/" rel="documentation"/>
+    <link href="https://www.ifao.egnet.net/uploads/publications/normes/IFAO_publications_normes_bibliographiques_pub_egypto_2020_fr.pdf" rel="documentation"/>
     <author>
-      <name>Alexandre Vincent</name>
-      <email>alexandre.vincent@univ-poitiers.fr</email>
-    </author>
-    <author>
-      <name>Nicolas Monteix</name>
-      <email>nicolas.monteix@univ-rouen.fr</email>
+      <name>Nicolas Souchon</name>
+      <email>souchon.nicolas.ns@gmail.com</email>
```

Comment:
@NicolasSouchon, thanks. I see you removed the "template" link and original style authors. Did you create your version from scratch, or did you based it on the existing IFAO style, or another CSL style?

Reply:
Yes, this style was created from an empty base using Visual CSL Editor. But I should add a link between this style and the [English version](https://github.com/citation-style-language/styles/pull/4870)

Reply:
I'm a little bit surprised you went for a complete rewrite. Did the IFAO change its citation format drastically recently? Our IFAO style was only contributed in April, so it's not very old.

@NMonteix, do you know if the style guide has recently changed, and whether this style would be an improvement over your version we currently have in the repository? See also #4870 for the English version of this new style.